docs/manuals/man1/pki.1.md

```diff
@@ -0,0 +1,350 @@
+pki 1 "February 1, 2019" PKI "pki CLI"
+======================================
+
+NAME
+----
+
+pki - Command-Line Interface for accessing PKI services.
+
+SYNOPSIS
+--------
+
+`pki` [CLI options] &lt;command&gt; [command arguments]
+
+DESCRIPTION
+-----------
+
+The `pki` command provides a command-line interface allowing clients
+to access various services on the PKI server.
+These services include certificates, groups, keys, security domains, and users.
+  
+CLI OPTIONS
+-----------
+
+`-c` &lt;NSS database password&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the NSS database password (mutually exclusive to the `-C` option).
+
+`-C` &lt;NSS database password file&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the file which contains the NSS database password (mutually exclusive to the `-c` option).
+
+`-d` &lt;NSS database location&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the NSS database location (default: *&#8764;/.dogtag/nssdb*).
+
+`-h` &lt;hostname&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Specifies the hostname (default: hostname of the local machine).
+
+`--help`  
+&nbsp;&nbsp;&nbsp;&nbsp;Prints additional help information.
+
+`--ignore-cert-status` &lt;list&gt;  
+&nbsp;&nbsp;&nbsp;&nbsp;Comma-separated list of ignored certificate validity statuses.
```

Comment:
Would be nice to have a pointer to the list of available statuses / where to find them.

Reply:
I added the values from this code:
https://github.com/dogtagpki/jss/blob/master/org/mozilla/jss/ssl/SSLCertificateApprovalCallback.java#L56